pkg/cmd/image/pull.go

```diff
@@ -99,6 +100,17 @@ func EnsureImage(ctx context.Context, client *containerd.Client, rawRef string,
 		if err != nil {
 			return nil, err
 		}
+	case "notation":
+		experimental := options.GOptions.Experimental
+
+		if !experimental {
```

Comment:
nit: if only used once, you can change to `if !options.GOptions.Experimental` directly